roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/RoundaboutModel.java

```diff
@@ -0,0 +1,56 @@
+package at.fhv.itm3.s2.roundabout;
+
+import desmoj.core.dist.ContDistUniform;
+import desmoj.core.simulator.Model;
+
+import java.util.Random;
+
+public class RoundaboutModel extends Model {
+
+    public static final double MIN_TIME_BETWEEN_CAR_ARRIVALS = 3.5;
+    public static final double MAX_TIME_BETWEEN_CAR_ARRIVALS = 10.0;
+
+    /**
+     * Random number stream used to draw a time between two car arrivals.
+     * See init() method for stream parameters.
+     */
+    private ContDistUniform timeBetweenCarArrivals;
+
+    /**
+     * Constructs a new RoundaboutModel
+     *
+     * @param model         the model this model is part of (set to null when there is no such model)
+     * @param name          this model's name
+     * @param showInReport  flag to indicate if this model shall produce output to the report file
+     * @param showInTrace   flag to indicate if this model shall produce output to the trace file
+     */
+    public RoundaboutModel(Model model, String name, boolean showInReport, boolean showInTrace) {
+        super(model, name, showInReport, showInTrace);
+    }
+
+    @Override
+    public String description() {
+        return null;
```

Comment:
maybe set something initial like "roundabout simulation model"

Reply:
good point will refactor it.